x/evm/keeper/keeper.go

```diff
@@ -82,14 +88,32 @@ func (k Keeper) CommittedCtx() sdk.Context {
 	return k.committedCtx
 }
 
+// Ctx returns the top of context stack
```

Comment:
"... if the stack is empty, it returns the committed state context"

Reply:
done